### PR TITLE
Template formaters

### PR DESCRIPTION
**Pull request by ranyefet**
Add support for registering custom formaters, that can be used in templates.
See other pull request for more info #1369 

@mdale @itaykinnrot  please review & merge

### CODE REVIEW

File: modules/KalturaSupport/UiConfResult.php [hunk 1]
```diff
@@ -241,6 +244,7 @@ private function walkThroughPluginAttributes( $attrArray ,&$refrencePathInObject
 				$this->walkThroughPluginAttributes( $attrValue , $refrencePathInObject[$attrKey] );
 			}
 			else {
+				if( in_array($attrKey, $this->propsToIgnoreWhenResolvingResource) ) continue;
 			    $refrencePathInObject[$attrKey] = $this->resolveCustomResourceUrl( $attrValue );
```
Comment (mdale) [inline modules/KalturaSupport/UiConfResult.php]: Why is this needed ? do templates resolve resource paths somewhere else?

**Comment by mdale** [comment]
@ranyefet looks fine; but documentation would be nice ;)

**Comment by itaykinnrot** [comment]
added to 2,27
